com.unity.render-pipelines.core/Editor/Gizmo/HierarchicalBox.cs

```diff
@@ -115,17 +115,27 @@ public Color baseColor
             set
             {
                 value.a = 8f / 255;
-                m_MonochromeFillColor = value;
-                material.color = m_MonochromeFillColor;
-                value.a = 1f;
-                m_MonochromeHandleColor = value;
-                value.a = 0.7f;
-                m_WireframeColor = value;
-                value.a = 0.2f;
-                m_WireframeColorBehind = value;
+                SetBaseColorWithoutIntensityChange(value);
             }
         }
 
+        /// <summary>
+        /// Set the baseColor used to fill hull. All other colors are deduced from it except specific handle colors.
+        /// Instead of <see cref="baseColor">baseColor</see> set, this will not force the opacity and keep what is provided for the filled faces.
+        /// </summary>
+        /// <param name="color">The color to use</param>
+        public void SetBaseColorWithoutIntensityChange(Color color)
```

Comment:
I usually prefer to have name that describes what does a function and not what it doesn't.

Maybe this one should be named only `SetBaseColor`